README.md

```diff
@@ -200,28 +200,19 @@ stack --stack-yaml=stack-8.0.2.yaml install
 
 ### Installation on Windows
 
-#### The `make` tool
+The `Makefile` doesn't work on Windows due to several UNIX-specific things, such
```

Comment:
Uhm, since when? I'm using makefile to build hie exlusively and everything works.

Reply:
I don't see how this could work: https://github.com/haskell/haskell-ide-engine/blob/master/Makefile#L22

When I build, there is no file `$(STACKLOCALBINDIR)/hie`, instead there is a file named `$(STACKLOCALBINDIR)/hie.exe` - note the extension. 

Could it be that you're running everything under Cygwin or under WSL or in Docker?

Reply:
@fsoikin no, i'm usually running it under cmd.exe but have msys binaries in path. The only thing i had to do is install make with msys.

Reply:
@Anrock so then, how does `cp $(STACKLOCALBINDIR)/hie $(STACKLOCALBINDIR)/hie-8.2.2` work for you? I'm assuming `cp` must come from `msys` (though I'm not sure it does), but what about the non-existent file `$(STACKLOCALBINDIR)/hie`?